tiled/authenticators.py

```diff
@@ -1,8 +1,11 @@
 import logging
```

Comment:
Since not every tiled instance will integrate with LDAP, would you consider moving these imports so that they are only done if an ldap authenticator is needed?

Reply:
It took a few tries there, but this is now fixed.